app/services/my_anime_list_sync_service.rb

```diff
@@ -0,0 +1,161 @@
+class MyAnimeListSyncService
+  ATARASHII_API_HOST = 'https://hbv3-mal-api.herokuapp.com/2.1/'.freeze
+  MINE = '?mine=1'.freeze
+
+  attr_reader :library_entry, :method
+
+  def initialize(library_entry, method)
+    @library_entry = library_entry
+    @method = method
+  end
+
+  def execute_method
```

Comment:
Can't do that within a service object.  We'll worry about that later, for now just `return` early to skip the sync